docs/devel/text-conventions.md

```diff
@@ -0,0 +1,60 @@
+# Text conventions
+
+The document describes the proper way to introduce new text to the Kubernetes Dashboard.
+
+## Overview
+Kubernetes Dashboard is a web-based user interface for Kubernetes clusters. It consists of multiple views like overview pages, list of resources and resource details pages. Much textual information is required within these elements. Let's consider sample dialog:
+
+![Dashboard dialog](../dashboard-ui.png)
+
+As you noticed there is a lot of different kinds of text messages in each view:
+
+- titles
```

Comment:
This seems to be a matter of style, but I've switched it. Actually I like to capitalize items in lists only if I'm using full sentences.